### DISCUSSION TITLE
Student ID

### DISCUSSION THREAD

**Discussion by george-ckito** [community/community#132438] [GitHub Education]
### Hi everyone,

my Student ID is a bit different. It shows the validity date on the back side, and my name and picture on the front side. I was wondering how to upload the both sides of my ID in my application.

**Comment by Bram1903** [comment]
Hey,

You can simply take two pictures and edit them together on your device. Otherwise try and upload only the front side of your ID (with your name and picture), as when using a verified school email address, if you have one, you probably don't even need the validity date to be present.

Otherwise, try and take a picture of both your student id, and something else that has your date of course or session.

Kind regards,
Bram

**Reply by george-ckito** [reply]
How do I check if my school email address is verified? It ends with @edu.ge

**Reply by Bram1903** [reply]
If you add your school email address as an extra email to your GitHub account and head over to the place where you apply for the education pack, it should look like this:

<img width="606" alt="Screenshot 2024-07-16 at 22 44 19" src="https://github.com/user-attachments/assets/7fe59a22-ca08-4c07-b02b-e3ff5bb767ab">
